src/components/index.js

```diff
@@ -26,7 +26,7 @@ export {
 } from './movie-displays/DisplayHelpers';
 
 // pages
-export { default as AboutPage } from './pages/AboutPage';
+export { default as AboutPage } from './pages/AboutPage/AboutPage';
```

Comment:
I think it makes sense to have this sub-folder, since `TeamMember` is exclusively for the `AboutPage`, but I you should stick with the name conventions we have just for clarity (i.e. caps case for components, kabob case for folders, etc).

Also, even though it isn't used anywhere else for now, I still think you should put `TeamMember` in this index file, just for consistency.

```suggestion
export { default as AboutPage } from './pages/about/AboutPage';
export { default as TeamMember } from './pages/about/TeamMember';
```